core/cpu.c

```diff
@@ -120,11 +120,11 @@ static uint32_t cpu_pop_word(void) {
     return value;
 }
 
-inline static uint8_t cpu_read_in(uint16_t pio) {
+__inline static uint8_t cpu_read_in(uint16_t pio) {
```

Comment:
Possibly. I don't think you'd be able to use ```inline``` as a macro though due to it being a reserved keyword in MSVC